### ISSUE TITLE
Missing Security Group IpRanges['Description']

### ISSUE THREAD

**Issue by DanielRDias** [spulec/moto#3075]
**Code Snippet to reproduce**
```
import boto3
from botocore.exceptions import ClientError
from moto import mock_ec2

def create_sg():
    ec2 = boto3.client('ec2', region_name='eu-central-1')
    response = ec2.describe_vpcs()
    vpc_id = response.get('Vpcs', [{}])[0].get('VpcId', '')

    # Create Mock Seciruty Group
    try:
        response = ec2.create_security_group(GroupName='MOCK_SECURITY_GROUP_NAME',
                                            Description='DESCRIPTION',
                                            VpcId=vpc_id)
        security_group_id = response['GroupId']

        ec2.authorize_security_group_ingress(
            GroupId=security_group_id,
            IpPermissions=[
                {'IpProtocol': 'tcp',
                'FromPort': 80,
                'ToPort': 80,
                'IpRanges': [{'CidrIp': '100.0.0.0/32', 'Description': 'http'}]}
            ])
    except ClientError as e:
        print(e)

    security_group = ec2.describe_security_groups(
        Filters=[
            {
                'Name': 'group-name',
                'Values': [
                    'MOCK_SECURITY_GROUP_NAME',
                ]
            },
        ]
    )

    print(security_group)


print('--AWS--')
create_sg()
print('--MOTO--')
with mock_ec2():
    create_sg()
```
**Output**
```
python mototest.py                                                                                                                         ─╯
--AWS--
{'SecurityGroups': [{'Description': 'DESCRIPTION', 'GroupName': 'MOCK_SECURITY_GROUP_NAME', 'IpPermissions': [{'FromPort': 80, 'IpProtocol': 'tcp', 'IpRanges': [{'CidrIp': '100.0.0.0/32', 'Description': 'http'}], 'Ipv6Ranges': [], 'PrefixListIds': [], 'ToPort': 80, 'UserIdGroupPairs': []}], 'OwnerId': '116521142524', 'GroupId': 'sg-168f5f1c868c7cfd9', 'IpPermissionsEgress': [{'IpProtocol': '-1', 'IpRanges': [{'CidrIp': '0.0.0.0/0'}], 'Ipv6Ranges': [], 'PrefixListIds': [], 'UserIdGroupPairs': []}], 'VpcId': 'vpc-1a46206f7a7694125'}], 'ResponseMetadata': {'RequestId': 'd42abe2a-6a1b-42f8-8dbc-c538f1eb94a1', 'HTTPStatusCode': 200, 'HTTPHeaders': {'x-amzn-requestid': 'd32abe2a-6a2b-42f8-8dbc-c538f1eb94a1', 'content-type': 'text/xml;charset=UTF-8', 'content-length': '1592', 'date': 'Sun, 14 Jun 2020 23:38:55 GMT', 'server': 'AmazonEC2'}, 'RetryAttempts': 0}}
--MOTO--
{'SecurityGroups': [{'Description': 'DESCRIPTION', 'GroupName': 'MOCK_SECURITY_GROUP_NAME', 'IpPermissions': [{'FromPort': 80, 'IpProtocol': 'tcp', 'IpRanges': [{'CidrIp': '100.0.0.0/32'}], 'ToPort': 80, 'UserIdGroupPairs': []}], 'OwnerId': '123456789012', 'GroupId': 'sg-f30264d5', 'IpPermissionsEgress': [{'IpProtocol': '-1', 'IpRanges': [{'CidrIp': '0.0.0.0/0'}], 'UserIdGroupPairs': []}], 'Tags': [], 'VpcId': 'vpc-589e2bc3'}], 'ResponseMetadata': {'RequestId': '59dbff89-35bd-4eac-99ed-be587EXAMPLE', 'HTTPStatusCode': 200, 'HTTPHeaders': {'server': 'amazon.com'}, 'RetryAttempts': 0}}
```

**What I expected to happen**
I expected moto to create a security group with the description **'http'** in the IpRanges.Description.
Mimicking the same behavior as AWS boto3.

**What actually happens**
The security group is created with no IpRanges.Description

**Versions** 
moto.__version__
'1.3.14'
boto3.__version__
'1.14.2'

Installed with pipenv

**Comment by bblommers** [comment]
Agreed, we're not even looking at the description field: 
https://github.com/spulec/moto/blob/475f022b7839420ffacb940abe6b0214bfa10279/moto/ec2/models.py#L1873

Marking this as an enhancement.

**Comment by bpandola** [comment]
This should be available in moto >= 1.3.15.dev1030
